src/OscService.js

```diff
@@ -0,0 +1,79 @@
+class OscServer {
```

Comment:
ok so this is kind of a weird thing - a Service is what gets returned from module.exports, but OscServer is kinda like `src/services/httpserver.js`. I am embedded a server and service together in the same file. Which unfortunately I cannot do with http/socketio because the socketio server depends on an httpserver TT

Reply:
There are three different styles of writing going on here, which is whats confusing
### services/osc
```js
class OscServer {} // define class inline
module.exports = { 
  create: client => {
    const server = new OscServer(client)
    server.listen(() => broadcast('osc'))
  }
}
```
### services/http
```js
module.exports = {
  create: client => {
    const HttpServer = require('./httpserver')
    const server = new HttpServer() // grab class externally
    server.context.client = client
    server.listen(() => broadcast('http')
  }
}
```
### services/socketio
```js
module.exports = {
  create: (client, { app }) => {
    const io = require('koa-socket')() // class defined not by us!
    io.context.client = app.context.client || client
    io.attach(app)
    app.listen(() => broadcast('socketio ', 'http'))
  }
}